controllers/cinder_controller.go

```diff
@@ -741,24 +748,25 @@ func (r *CinderReconciler) reconcileNormal(ctx context.Context, instance *cinder
 			Log.Info(fmt.Sprintf("Deployment %s successfully reconciled - operation: %s", instance.Name, string(op)))
 		}
 
-		// Mirror CinderBackup status' ReadyCount to this parent CR
-		instance.Status.CinderBackupReadyCount = cinderBackup.Status.ReadyCount
+		// Only if the generation of the CinderBackup is the same as the observed generation, can we mirror the condition and ready count
```

Comment:
This comment is worded very different from the one on L691.

Reply:
Changed.